packages/react-native-reanimated/android/src/main/java/com/swmansion/reanimated/NodesManager.java

```diff
@@ -456,25 +455,17 @@ public String obtainProp(int viewTag, String propName) {
         Drawable background = view.getBackground();
         int actualColor = -1;
 
-        // ReactViewBackgroundDrawable got deprecated in react-native 0.75.
-        //noinspection deprecation
-        if (background instanceof ReactViewBackgroundDrawable) {
-          //noinspection deprecation
-          actualColor = ((ReactViewBackgroundDrawable) background).getColor();
-        }
+        try {
+          Method getColor = background.getClass().getMethod("getColor");
+          actualColor = (int) getColor.invoke(background);
```

Comment:
It can be scope variable
```suggestion
          int actualColor = (int) getColor.invoke(background);
```